### DISCUSSION TITLE
How to build

### DISCUSSION THREAD

**Discussion by Cassy-Lee** [qdm12/ddns-updater#753] [Q&A] [answered]
How to build a standalone binary? I can't use docker and last release is four month ago.

**Comment by qdm12** [comment, marked as answer]
5b800cf278037b86c529e52da133ee569a9633d5 adds instruction to build the latest version

> You can alternatively download, build and install the latest version of the program by installing [Go](https://golang.org/doc/install) and then run `go install github.com/qdm12/ddns-updater/cmd/ddns-updater@latest`

[v2.7.0 release](https://github.com/qdm12/ddns-updater/releases/tag/v2.7.0) with pre-built binaries 😉
